packages/devtools_app/lib/src/shared/diagnostics/variable_factory.dart

```diff
@@ -344,8 +365,25 @@ List<DartObjectNode> createVariablesForAssociations(
     false,
     (p, e) => p || isPrimativeInstanceKind(e.key.kind),
   );
-  for (var i = 0; i < associations.length; i++) {
+  for (final i in Iterable.generate(associations.length)) {
```

Comment:
what's the reason for this change? This requires us to create another iterable instead of just looping through the one we already have `associations`

Reply:
reverted